frontend/src/views/web/challenge/delete-challenge/delete-challenge.html

```diff
@@ -0,0 +1,23 @@
+<section class="text-center pd-20 width-500">
+    <div class="row ht-wt-100p">
+        <div class="col s12 m12">
+            <div class="ev-card-body">
+                <form name="deleteChallengeForm" ng-submit="challenge.deleteChallenge(deleteChallengeForm.$valid)">
+                    <div class="pass-title">Delete Challenge</div>
+                    <div class="input-field align-left">
+                        <input placeholder="Enter challenge name" type="text" ng-model="nameInput">
+                    </div>
+                    <ul class="inline-list hide-on-med-and-down">
+                        <li>
+                            <a ng-click="challenge.deleteChallenge(false)" class="dark-link pointer"><strong>Cancel</strong></a>
+                        </li>
+                        <li>
+                            <button class="btn red waves-effect waves-dark grad-btn darken-2 fs-14" ng-disabled="!(nameInput==challenge.page.title)" type="submit" value="submit">Confirm</button>
```

Comment:
Please change the text with `I understand consequences, delete the challenge.`

Reply:
@deshraj Done
![screenshot from 2018-02-07 01-08-49](https://user-images.githubusercontent.com/26835119/35880274-8069d0ac-0ba3-11e8-9d0a-c5053108278d.png)